aergo/herapy/obj/sc_state.py

```diff
@@ -34,6 +34,9 @@ def get_storage_key(self):
     def __str__(self):
         return self.storage_key
 
+    def __bytes__(self):
+        return self.storage_key.encode('latin-1')
```

Comment:
Would you tell me why it uses 'latin-1' instead of 'utf-8'?

Reply:
the purpose it to enable querying the real bytes of the key so since SCStateWar.var_name is a string, we can do so with var_name="\x90" to access b'\x90' of the trie key. But b'\x90' cannot decode to 'utf-8' : 
`"\x90".encode('utf-8') >>> b'\xc2\x90'`  => this is not the real bytes, expected b'\x90'
`"\x90".encode('latin-1') >>> b'\x90'`

Reply:
Got it! Thanks :)